src/main/webapp/components/directives/cache/loaders/cache-loader.js

```diff
@@ -178,8 +174,9 @@
         scope.fieldValueModified = function (field) {
           var original = scope.prevData[field];
           var latest = scope.data[field];
+          var customLoaderSelected = field === 'class' && latest === '' && utils.isNullOrUndefined(original);
 
-          if (((utils.isNullOrUndefined(original) || original === '') && !latest) || original === latest) {
+          if (!customLoaderSelected && ((utils.isNullOrUndefined(original) || original === '') && !latest) || original === latest) {
```

Comment:
@vblagoje Good point, I'll have another look and try to simplify it and/or break it down into multiple functions.